### PR TITLE
[MIG] stock_by_warehouse_sale: Migration to 13.0

### PR DESCRIPTION
**Pull request by agarzaarvizu**
- Migration of the module stock_by_warehouse_sale to 13.0

### CODE REVIEW

**Comment by agarzaarvizu** [comment]
@hugho-ad Please review.

**Review by luisg123v — CHANGES_REQUESTED** [review]
@agarzaarvizu there's only one commit, remember to keep history so we may know what changes are being performed to migrate.

**Comment by agarzaarvizu** [comment]
@hugho-ad Done. Please review

**Comment by hugho-ad** [comment]
LGTM :+1:

File: stock_by_warehouse_sale/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_by_warehouse_sale
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0+e\n"
```
Comment (luisg123v) [inline stock_by_warehouse_sale/i18n/es.po]: 13

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: stock_by_warehouse_sale/i18n/es.po (hunk 1 above)
Comment (agarzaarvizu) [reply stock_by_warehouse_sale/i18n/es.po]: @luisg123v Done

**Review by luisg123v — APPROVED** [review]

**Pull request merged**